sdv/constraints/tabular.py

```diff
@@ -292,22 +325,31 @@ def reverse_transform(self, table_data):
         """
         table_data = table_data.copy()
         diff = (np.exp(table_data[self._diff_column]).round() - 1).clip(0)
-        if self._diff_is_datetime(table_data):
+        if self._is_datetime:
```

Comment:
Could `_is_datetime` ever be `None` here? Since it depends on calling `fit` first.

Reply:
it could. In a similar way though, `self._diff_column` could also be `None`. So I am assuming we want users to fit the constraint before reverse transforming. Not sure what you think

Reply:
I see. If it becomes a usability problem, we could add an explicit check and throw an informative error. Seems fine for now since most constraints probably assume this.

Reply:
I think if we did that, we should probably just have on in the base class that complains if you try and reverse transform or transform before having fit anything